cheatsheets/Server_Side_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -287,18 +287,18 @@ The first validation on the input data presented in the case [n°1](Server_Side_
+1. The application will receive the IP address or domain name of the *TargetedApplication* and it will apply the first validation on the input data using the libraries/regex mentioned in this [section](Server_Side_Request_Forgery_Prevention_Cheat_Sheet.md#application-layer).
+2. The second validation will be applied against the IP address or domain name of the *TargetedApplication* using the following blacklist approach:
     * For IP address: 
-        * The application verify that it is a public one (see the hint provided in the next paragraph with the python code sample).
+        * The application will verify that it is a public one (see the hint provided in the next paragraph with the python code sample).
     * For domain name: 
-        1. We verify that it is a public one by trying to resolve the domain name against the DNS resolver that only resolve internal domain name. Here, it must return a response indicating that it do not know the provided domain because the expected value received must be a public domain.
-        2. To prevent the *DNS pinning* attack described into this [document](../assets/Server_Side_Request_Forgery_Prevention_Cheat_Sheet_SSRF_Bible.pdf), we retrieve the IPs behind the domain name provided (taking records *A* + *AAAA* for IPv4 + IPv6) and we apply the same verificaton than if we have received an IP address.
-3. The application receive the protocol to use for the request via a dedicated input parameter for which we verify the value against a allowed list of protocols (`HTTP` or `HTTPS`).
-4. The application receive the parameter name for the token to pass to the *TargetedApplication* via a dedicated input parameter for which we only allow characters set `[a-z]{1,10}`.
-5. The application receive the token itself via a dedicated input parameter for which we only allow characters set `[a-zA-Z0-9]{20}`.
-6. The application receive and validate (from a security point of view) any business data needed to perform a valid call.
-7. The application build the HTTP POST request **using only validated informations** and send it (*we do not forget to disable the support for [redirection](https://developer.mozilla.org/en-US/docs/Web/HTTP/Redirections) in the web client used*).
+        1. The application will verify that it is a public one by trying to resolve the domain name against the DNS resolver that will only resolve internal domain name. Here, it must return a response indicating that it do not know the provided domain because the expected value received must be a public domain.
+        2. To prevent the `DNS pinning` attack described into this [document](../assets/Server_Side_Request_Forgery_Prevention_Cheat_Sheet_SSRF_Bible.pdf), the application will retrieve all the IP adddresses behind the domain name provided (taking records *A* + *AAAA* for IPv4 + IPv6) and it will apply the same verification described in the previous point about IP addresses.
```

Comment:
- `in` instead of `into`
- `addresses` instead of `adddrresses`

Reply:
Fixed for `into` but  `adddrresses` do not exists into the document:
![Capture](https://user-images.githubusercontent.com/1573775/60345084-79eb3280-99b8-11e9-9472-6cbd1fa05d48.png)